kafka-ui/app.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: &app kafka-ui
+  labels:
+    app.kubernetes.io/name: *app
+spec:
+  ports:
+    - name: http
+      port: 8080
+  selector:
+    app.kubernetes.io/name: *app
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: &app kafka-ui
+  annotations:
+    "app.uw.systems/tags.oss": "true"
+    "app.uw.systems/tier": "tier_3"
+    "app.uw.systems/description": "UI for interacting with the Kafka Cluster using https://docs.kafka-ui.provectus.io"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: *app
+  template:
+    metadata:
+      labels:
+        app: *app
+        app.kubernetes.io/name: *app
+        policy.semaphore.uw.io/name: "kafka-ui"
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+        - name: *app
+          imagePullPolicy: Always
+          image: provectuslabs/kafka-ui:v0.7.1
+          env:
+            # We need to list here all the config files loaded in the kafka-ui-config ConfigMap.
+            # Although Spring Boot should support just setting the value to the folder "file:/kafka-ui-config/" as stated here: https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.files.optional-prefix,
+            # the config files are not picked up when the value is set like that
```

Comment:
Late to the party, just having a read around what's going on.
Is this use-case not covered with https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.files.configtree i.e. with `configtree:`?